ResourceManager/GitExtensionsControl.cs

```diff
@@ -119,6 +119,28 @@ public virtual bool ProcessHotkey(Keys keyData)
             return hotkey is not null && ExecuteCommand(hotkey.CommandCode);
         }
 
+        /// <summary>
+        /// Runs the associated command, bubbling up the tree if this control does not handle it.
+        /// This mimicks the behavior of <cref>ProcessCmdKey</cref>, in allowing the command request to bubble.
+        /// </summary>
+        public bool DispatchCommand(int command)
+        {
+            for (Control control = this; control != null; control = control.Parent)
+            {
+                if (control is GitExtensionsControl extensionsControl && extensionsControl.ExecuteCommand(command))
+                {
+                    return true;
+                }
+
+                if (control is GitExtensionsFormBase extensionsForm && extensionsForm.DispatchCommand(command))
+                {
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
```

Comment:
Move to `GitExtensionsControl.ExecuteCommand`, please.

Reply:
Not possible. See above.